src/Router.tsx

```diff
@@ -20,6 +22,11 @@ export default function Router() {
         <Route path="/create">
           <V1Create />
         </Route>
+        <Route path="/create-v2">
+          <V2UserProvider>
```

Comment:
Eventually we'll probably move this to `Router.tsx`. For now, let's keep it as scoped as possible to minimize risk of leaks.